src/main/java/io/vertx/core/file/AsyncFile.java

```diff
@@ -181,4 +181,30 @@
    */
   @Fluent
   AsyncFile setReadBufferSize(int readBufferSize);
+
+  /**
+   * Like {@link #size()} but blocking.
+   *
+   * @throws FileSystemException if an error occurs
+   */
+  default long sizeBlocking() {
```

Comment:
Why a default here? It seems to be implemented on the impl class.

Reply:
Same question as Paulo :)

Reply:
I removed the default implementations. PTAL